docs/architecture/index.md

```diff
@@ -17,9 +17,9 @@ There are a number of technologies or functional design choices that provide hig
 
 The `kube-vip` service builds a multi-node or multi-pod cluster to provide High-Availability. When a leader is elected, this node will inherit the Virtual IP and become the leader of the load-balancing within the cluster. 
 
-When running **out of cluster** it will use [raft](https://en.wikipedia.org/wiki/Raft_(computer_science) clustering technology 
+When using ARP or layer2 it will use [leader election](https://godoc.org/k8s.io/client-go/tools/leaderelection)
 
-When running **in cluster** it will use [leader election](https://godoc.org/k8s.io/client-go/tools/leaderelection)
+It is also possible to use [raft](https://en.wikipedia.org/wiki/Raft_(computer_science) clustering technology, but this approach has largely been superseded by leader election.
```

Comment:
This implies that it is deprecated. Is anyone using it? Do you have a need to maintain it? If not, maybe just deprecate it explicitly?

Reply:
It's still a requirement to run a Load-Balancing cluster without Kubernetes.